### DISCUSSION TITLE
[Esame di prova 18 dicembre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#425] [Esami M-Z]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/597f3c3e-9c8d-4058-af72-3a4ff735698c)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/f181fea1-a112-43d3-96dd-3fc5a9c922e8)
